File: fridaload/src/main/java/com/hepta/fridaload/MainActivity.java
package com.hepta.fridaload;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {



    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button btn = findViewById(R.id.loadfrida);
        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                new Thread(){
                    @Override
                    public void run() {
//                        LoadEntry.Entry(getApplicationContext(),getApplicationInfo().sourceDir);
//                        System.loadLibrary("frida-gadget");
                        LoadEntry.text("rerewrewrew");
                    }
                }.start();
            }
        });

    }


}

File: dev/src/main/java/com/example/heptalinker/MainActivity.kt
package com.example.heptalinker

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView
import com.example.heptalinker.databinding.ActivityMainBinding
import com.hepta.linker.NativeLib
class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
        val lib = NativeLib()
        // Example of a call to a native method
        binding.sampleText.text = lib.stringFromJNI()
    }

}